backend/danswer/connectors/document360/connector.py

```diff
@@ -0,0 +1,130 @@
+import requests
+from datetime import datetime, timezone
+from typing import Any, Generator, Optional, List, Union
```

Comment:
Looks like you might be a consistent contributor! Happy to see that!

Please follow the [CONTRIBUTING](https://github.com/danswer-ai/danswer/blob/main/CONTRIBUTING.md) guide to make sure your code is linted and type checking passes

Linter should take care of restructuring the imports. The Mypy issues need to be addressed manually though. Please address those, thanks!

Reply:
Okay, I fixed type checking issue.